modules/apps/callbacks/types/keys.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+type CallbackTrigger string
```

Comment:
Not a huge fan of `Trigger` as a coding term. I was happy enough with `Type`, but no strong preference. Thanks for updating the other names though!

Reply:
I was thinking of using CallbackType for `"source_callback"` and `"destination_callback"`. I'm refactoring some stuff related to events and logs, if I end up not needing this (seems more likely now), I'll revert it.

Reply:
reverted